scss/_base_button.scss

```diff
@@ -44,8 +44,9 @@
     @media only screen and (min-width: $breakpoint-x-small + 1) {
       width: auto;
 
-      &:not(:last-of-type):not(:only-of-type) {
-        margin-right: $sph-outer;
+      + [class^='p-button'],
+      + [class*='p-button'] {
```

Comment:
Why both `^=` and `*=` are needed?
And I guess this will not cover `<button /> <button />` case.

Reply:
True - fixing